jbmc/unit/java_bytecode/java_bytecode_convert_method/convert_method.cpp

```diff
@@ -308,6 +308,34 @@ class java_bytecode_convert_method_unit_testt
   {
     return converter.convert_astore(statement, op, location);
   }
+
+  static exprt variable(
+    java_bytecode_convert_methodt &converter,
+    const exprt &arg,
+    char type_char,
+    size_t address,
+    bool cast_as_needed)
+  {
+    return converter.variable(
+      arg,
+      type_char,
+      address,
+      cast_as_needed ? java_bytecode_convert_methodt::CAST_AS_NEEDED
+                     : java_bytecode_convert_methodt::NO_CAST);
+  }
+
+  static void add_variable(
```

Comment:
:bulb: Since isn't just a wrapper around `java_bytecode_convert_methodt` might be worth documenting

Reply:
With the refactoring that removes the extra argument, it is now just a trivial wrapper